rsts/community/roadmap.rst

```diff
@@ -71,7 +72,7 @@ Issues by Components
 +--------------+-----------------------------------------------+-----------------------------------------------------------------------------+--------------------------------------------+
 | Theme        | Description                                   | Open Issues                                                                 | Comment                                    |
 +--------------+-----------------------------------------------+-----------------------------------------------------------------------------+--------------------------------------------+
-| flyteconsole | Issues on FlyteConsole (Flytes UI)            | `flyteconsole issues <https://github.com/flyteorg/flyte/labels/ui>`_        | These are great issues to get started with |
-+--------------+-----------------------------------------------+-----------------------------------------------------------------------------+--------------------------------------------+
-| flytectl     | Issues on flytectl (standalone CLI for flyte) | `flytectl issues <https://github.com/flyteorg/flyte/labels/flytectl>`_      | Great issues to start with                 |
+| flyteconsole | Issues on FlyteConsole (Flytes UI).           | `flyteconsole issues <https://github.com/flyteorg/flyte/labels/ui>`_        | These are great issues to get started with.|
 +--------------+-----------------------------------------------+-----------------------------------------------------------------------------+--------------------------------------------+
+| flytectl     | Issues on FlyteCTL (standalone CLI for Flyte) | `FlyteCTL issues <https://github.com/flyteorg/flyte/labels/flytectl>`_      | Great issues to start with.                |
++--------------+-----------------------------------------------+-----------------------------------------------------------------------------+--------------------------------------------+
```

Comment:
Can we add this in the end?

There's a :ref:`live roadmap <https://github.com/orgs/flyteorg/projects/3>`__ we highly encourage you to look at to get a consolidated view of what we're working on currently.